### PR TITLE
Making it easier for our users to report documentation issues or questions

### PR DESCRIPTION
**Pull request by kaushikvelusamy**
This pointer on the left pane will speedup the process to find documentation issues / questions and help resolve documentation faster by quickly assigning the relevant docs section owner.

### CODE REVIEW

File: docs/issues.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Questions/Issues on ALCF Docs
+
+Our ALCF documentations are hosted through GitHub to make it easier to contribute and collaborate.
```
Comment (felker) [inline docs/issues.md:3]:
```suggestion
The ALCF documentation source code is hosted on GitHub to make it easier to contribute and collaborate.
```

File: docs/issues.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
+
+If you find any issues on our documentation page, raise an issue at [ALCF Github user guide webpage](https://github.com/argonne-lcf/user-guides/issues/new). Provide us with a brief description on the documentation issue, page url, section heading and mark the appropriate label type on the right pane.
```
Comment (felker) [inline docs/issues.md:6]:
```suggestion
If you find any issues within these pages, please open an Issue at [ALCF GitHub user guide webpage](https://github.com/argonne-lcf/user-guides/issues/new). Provide us with a brief description of the issue, page URL, section heading, and mark the appropriate label type on the right pane. 
```

**Review by felker — CHANGES_REQUESTED** [review]

**Review by kaushikvelusamy — COMMENTED** [review]
Accepted all suggestions.

**Review by felker — APPROVED** [review]
LGTM, but have others already weighed in on this change? I think it is a good idea, generally, but maybe we would rather put it on the home page like NERSC does?

image


https://docs.nersc.gov/

**Comment by kaushikvelusamy** [comment]
I think we should have a pointer to this on every documentation page making it structurally easier for users to quickly raise issues and contribute faster. 

We should also add a guideline preamble on the new issue page to triage and delegate it to the right person faster . Example: https://github.com/pytorch/pytorch/issues/new?assignees=&labels=&projects=&template=documentation.yml

**Pull request merged**